src/commons/UI/BaseLabelAndTextInput.vue

```diff
@@ -0,0 +1,133 @@
+<template>
+  <div class="flex flex-col">
+    <label :for="inputName" :class="labelClass"
+      >{{ label }}<span v-if="isRequired" class="text-red-5"> *</span></label

```

Comment:
This looks strange, can we move <label... to the next line.

Reply:
Due to prettier krub.